apps.toml

```diff
@@ -2013,6 +2013,11 @@ state = "notworking"
 subtags = [ "pastebin" ]
 url = "https://github.com/YunoHost-Apps/modernpaste_ynh"
 
+[moncycle.app]
+category = "small_utilities"
+state = "working"
```

Comment:
it seems that the master branch is far from working !!!!!

Reply:
(the testing one seems more promising)

Reply:
Well, I have it working on my test server and have checked manually all scenarii (upgrade, restore, backup, install, remove, change url public or not...). 
I can put it as in progress if you prefer, but then I'll have to make a new PR again once the CI will have run on my current PR to testing branch.

Reply:
You should only put the state as working when you have merged.
Because for now it will add in the catalogue apps.json an app with the id "example" and it will launch a package_check on that app

Reply:
Ok, sorry for the bad timing. I didn't expect to wait for the CI so long. I'll ping you when everything will be validated and merged, thanks 🙏